stake-monitor/README.md

```diff
@@ -0,0 +1,14 @@
+## Overview
+`solana-stake-monitor` is a utility that scans all transactions to ensure that stake accounts remain in compliance with the following rules:
+
+1. The stake account must be created after genesis
+1. The "compliant balance" of a stake account is set upon stake account initialization, system transfers of additional funds into a compliant stake account are excluded from the "compliant balance"
+1. The stake account cannot have a lockup or custodian
+1. Withdrawing funds from the stake account trigger non-compliance
+1. Stake accounts split from a compliant stake account remain compliant, and the "compliant balance" is adjusted accordingly for the original stake account
+
+In terms of `solana` command-line subcommands:
+* `create-stake-account`: Creates a compliant stake account provided the `--lockup-date`, `--lockup-epoch`, or `--custodian` options are not specified
+* `delegate-stake` / `deactivate-stake` / `stake-authorize` / `split-stake`: These commands do not affect compliance
```

Comment:
Does the result of `split-account` get monitored?  And does a withdraw to that account bubble up to invalidate its parent(s)?

Reply:
> Does the result of split-account get monitored? 

Yes, the split account is monitored independently.

> And does a withdraw to that account bubble up to invalidate its parent(s)?

No, it doesn't.   But this business logic could be encoded in the 90% stake guarantee program that's built on top of this information.  There'll need to be a user workflow to re-register the pubkeys for the new split accounts -- either at registration or redemption time.  This gets complicated, which is why I initially wanted to ban split entirely.   I think enough information about the `stake-split` is  retained in the `solana-stake-monitor` data file that we could implement a variety of options